src/evm/transaction-count.ts

```diff
@@ -5,18 +5,14 @@ import { SponsorWalletTransactionCount } from '../types';
 import { retryGo } from '../utils';
 import { deriveSponsorWallet, shortenAddress } from '../wallet';
 
-// TODO: should this be in a centralized enum somewhere (api3/airnode-protocol maybe)?
-const pspProtocolId = '2';
-
 export const getSponsorWalletAndTransactionCount = async (
   airnodeWallet: ethers.Wallet,
   provider: ethers.providers.Provider,
   currentBlock: number,
   sponsor: string
 ): Promise<node.LogsData<SponsorWalletTransactionCount | null>> => {
   // Derive sponsorWallet address
-  // TODO: switch to node.evm.deriveSponsorWallet when @api3/airnode-node allows setting the `protocolId`
-  const sponsorWallet = deriveSponsorWallet(airnodeWallet.mnemonic.phrase, sponsor, pspProtocolId).connect(provider);
+  const sponsorWallet = deriveSponsorWallet(airnodeWallet.mnemonic.phrase, sponsor, '2').connect(provider);
```

Comment:
This is the `2` mentioned [here](https://api3workspace.slack.com/archives/C02AYRX8D89/p1645698379511399), right?
I think that should be stored in one place as you've said in the comment. Why did you decide not to go that way?

Reply:
I assumed no one really cared about that suggestion and that's why I just hardcoded `'2'` 😁  Also node package has '1' hardcoded. Maybe we should just create another issue for moving those protocol ids to some common place?

Reply:
You're right, I've created an [issue](https://api3dao.atlassian.net/browse/AN-629).